screens/Home/MyVcs/IdInputModal.tsx

```diff
@@ -80,6 +97,25 @@ export const IdInputModal: React.FC<IdInputModalProps> = props => {
                   selectionColor={Theme.Colors.Cursor}
                   value={controller.id}
                   keyboardType="number-pad"
+                  rightIcon={
+                    <Tooltip
```

Comment:
Can we also extract this Tooltip to a component similar to tooltipContent?

Reply:
Also if we having similar customisations for toolTip in other components as well (eg - GetIdInputModal) , can we create a ToolTip component in components/ui similar to Text component?
This way we can maintain consistency in our customization with Tooltip & avoid duplication

Reply:
Created an custom tootip component in ui